RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -1022,6 +1025,14 @@ effect[int] effectList();
 int[int] intList();
 item[int] itemList();
 
+########################################################################################################
+//Defined in autoscend/auto_monsterparts.ash
+boolean hasArm(monster enemy);
+boolean hasHead(monster enemy);
+boolean hasLeg(monster enemy);
+boolean hasTail(monster enemy);
+boolean hasTorso(monster enemy);
```

Comment:
I just had a look at this absolute monstrosity and it made me sad. Not only are the majority of these functions commented out and therefore not actually implemented, they're hardcoded lists. I think mafia has tracking for body parts somewhere, although I'm not sure. This is definitely something we should look in to rewriting.

Reply:
Very much agreed